app/assets/javascripts/app/payment.js

```diff
@@ -42,7 +42,13 @@ App.addChild('Payment', _.extend({
   },
 
   selectNationalPayment: function() {
-    this.onTabClick({currentTarget: this.$('#MoIP')});
+    $tab_target = $('#payment-methods').data('preferred-engine');
+
+    if($tab_target == '#') {
+      $tab_target = $('#MoIP');
+    }
+
+    this.onTabClick({currentTarget: this.$($tab_target)});
```

Comment:
Isso deve forçar o clique à tab `#MoIP` ou `#MetodoDePagamentoPreferido`. Eu tentei usar `this.$('#' + tabTarget).trigger( "click" );`, mas isso fazia com que o `loading image` não parasse. Tem alguma sugestão?